### DISCUSSION TITLE
Changing email parameter in /etc/sympa/sympa.conf

### DISCUSSION THREAD

**Discussion by UAProfessorProton** [sympa-community/sympa#1891] [Q&A]
We are attempting to change the e-mail address that our Sympa system, will use to accept MAIL commands.  We found the following the following in the sympa.conf documentation in changing the email parameter

![image](https://github.com/user-attachments/assets/3d1939e9-eaab-4cfc-8d64-2d371f4e4eac)

One line mentioned:
"If you change the default value, you must modify the mail aliases too." - We use a backend MySQL database for aliases that our MTA searches for when receiving mails, does this mean the alias should be added there as well? Sorry for the noob question.  Can we get more information regarding on how Sympa checks for mail aliases?  Thank you.

**Comment by ikedas** [comment]
Hey @UAProfessorProton , could you please share with us how you solved your question?

**Reply by UAProfessorProton** [reply]
Our organization's Postfix service use an mysql database backend for e-mail aliases. Since we need to use list@example.com and not sympa@example.com as Sympa's email address for MAIL commands, we added an alias entry in the database where list@example.com is just really an alias for sympa@example.com. This way, it didn't require for us to change the email parameter in /etc/sympa/sympa.conf.
